### DISCUSSION TITLE
We just got FEATURED 🚀👀

### DISCUSSION THREAD

**Discussion by nayakastha** [dscnitrourkela/project-avocado#207] [Announcements]
## We proudly announce that PROJECT-AVOCADO becomes our first endeavour to be recognized by GOOGLE DEV LIBRARY. Built with collaboration with ICS NITR, it is our longest-running project. Check it out [here](https://bit.ly/icsAppFeatured) ##

![Featured](https://user-images.githubusercontent.com/58568514/130592116-91a57f3a-153c-441a-b74e-1efb1f7a9ebd.jpeg)

**Check out the project and 🌟 the repository. To all the _Flutter Developers_ out there, contribute and learn.**

**Comment by abhibhaw** [comment]
Let's add some more achievements here
